### ISSUE TITLE
A helper named `unboundIf` could not be found

### ISSUE THREAD

**Issue by polesen** [jerel/ember-cli-growl#4]
Simply installed, put the stuff in application.hbs and growling something gave me this error.
I am running ember-cli-growl 1.0.2 with ember ember-cli 0.2.3 and ember 1.11.1.

```
Uncaught Error: Assertion Failed: A helper named `unboundIf` could not be found
Ember.default.assert @ ember.debug.js:5162
block @ ember.debug.js:7199
render @ growl-instance.js:340
renderHTMLBarsTemplate @ ember.debug.js:7845
renderView @ ember.debug.js:7812
renderView @ ember.debug.js:36758
mixin.Mixin.create.render @ ember.debug.js:36779
EmberRenderer_createElement @ ember.debug.js:38903
Renderer_renderTree @ ember.debug.js:8435
merge.default.ensureChildrenAreInDOM @ ember.debug.js:40205
View.default.extend._ensureChildrenAreInDOM @ ember.debug.js:40168
Queue.invoke @ ember.debug.js:871
Queue.flush @ ember.debug.js:936
DeferredActionQueues.flush @ ember.debug.js:741
Backburner.end @ ember.debug.js:166
Backburner.run @ ember.debug.js:221
run @ ember.debug.js:15925
handleRegisteredAction @ ember.debug.js:18434
(anonymous function) @ ember.debug.js:38121
jQuery.event.dispatch @ jquery.js:4665
jQuery.event.add.elemData.handle @ jquery.js:4333
```

**Comment by jerel** [comment]
To fix override the `growl-instance.hbs` template and change to regular `{{#if foo}}` blocks. I'll try to get a new release out shortly

**Comment by polesen** [comment]
Thanx. That helped :)

**Comment by jerel** [comment]
Closed by da82ff6ce2ea0d57f70b9124c8f7ab6fe0a35713
